app/frontend/src/components/Navigation.tsx

```diff
@@ -1,113 +1,162 @@
+      duration: theme.transitions.duration.short,
+    }),
+    padding: '6px 12px 8px',
+    minWidth: 80,
+    maxWidth: 168,
+    color: theme.palette.text.secondary,
+    flex: '1',
   },
+  selected: {
+    color: theme.palette.primary.main,
+  }
 }));
 
 export default function Navigation() {
   const classes = useStyles();
 
   const user = useTypedSelector((state) => state.auth.user);
-  const [open, setOpen] = useState(false);
-
-  const handleClick = () => {
-    setOpen(!open);
-  };
-
-  const menu = (
-    <List className={classNames(classes.listContainer, classes.menuList)}>
-      <ListItem button component={Link} to="/" className={classes.item}>
-        <ListItemText>Dashboard</ListItemText>
-      </ListItem>
-      <ListItem
-        button
-        component={Link}
-        to={profileRoute}
-        className={classes.item}
-      >
-        <ListItemText>Profile</ListItemText>
-      </ListItem>
-      <ListItem
-        button
-        component={Link}
-        to={messagesRoute}
-        className={classes.item}
-      >
-        <ListItemText>Messages</ListItemText>
-      </ListItem>
-      <ListItem
-        button
-        component={Link}
-        to={requestsRoute}
-        className={classes.item}
-      >
-        <ListItemText>Requests</ListItemText>
-      </ListItem>
-      {user ? (
-        <ListItem
-          button
-          component={Link}
-          to={logoutRoute}
-          className={classes.item}
-        >
-          <ListItemText>Logout</ListItemText>
-        </ListItem>
-      ) : null}
-      <SearchBox />
-    </List>
-  );
 
-  //this puts the navigation under a collapsable button on mobile only
   return (
-    <nav>
-      <Hidden mdUp>
-        <ListItem
-          button
-          component="a"
-          onClick={handleClick}
-          className={classes.item}
+    <AppBar
+      position="fixed"
+      classes={{
+        root: classes.appBar
+      }}
+      color="inherit"
+    >
+      <Toolbar
+        classes={{
+          gutters: classes.gutters
+        }}
+      >
+        <Hidden smDown>
+          <Typography 
+            variant="h5" 
+            className={classes.title}
+          >
+            Couchers
+          </Typography>
+        </Hidden>
+        <Grid
+          container
+          wrap="nowrap"
+          classes={{
+            root: classes.flex
+          }}
         >
-          <ListItemText>Navigation</ListItemText>
-        </ListItem>
-        <Collapse in={open} timeout="auto">
-          {menu}
-        </Collapse>
-      </Hidden>
-
-      <Hidden smDown>{menu}</Hidden>
-    </nav>
+        {menu.map(item => (
+          <Button
+            activeClassName={classes.selected}
+            classes={{
+              root: classes.item,
+              label: classes.label
+            }}
+            component={NavLink}
+            exact
+            to={item.route}
+            key={item.name}
+          >
```

Comment:
Not sure if `NavLink` with `activeClassName` is the best way to know/show active route.

Reply:
Seems good